app/models/feed.rb

```diff
@@ -75,6 +77,16 @@ def sanitize_text
     self.title = sanitize_text_only(title) if title_changed?
   end
 
+  def set_public_feeds_url
+    if public? && url.blank? && ENV["PUBLIC_FEEDS_PREFIX"].present?
+      self.url = "#{ENV["PUBLIC_FEEDS_PREFIX"]}/#{podcast.path}/#{published_path}"
+    end
+  end
```

Comment:
Note this is only set if the feed is public _and_ you left the `Public Feed URL` field blank.

And on deploy, this won't do anything until we set that ENV.